vultr/downloader.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# https://docs.vultr.com/vultr-ip-space
+# https://geofeed.constant.com/?json
+
+set -euo pipefail
+set -x
+
+
+# get from public ranges
+curl -s https://geofeed.constant.com/?json > /tmp/vultr.json
```

Comment:
Maybe download text: https://geofeed.constant.com/?text and split by `grep` (as another downloaders here)?

Reply:
Will try

Reply:
should be fine now